### ISSUE TITLE
Unknown password

### ISSUE THREAD

**Issue by raahimfareed** [pajt/game#1]
Kindly tell password to zip

**Comment by kixxu** [comment]
Man, it's virus. DON'T DOWNLOAD IT.

**Comment by raahimfareed** [comment]
> Man, it's virus. DON'T DOWNLOAD IT.

Yeah I know, I'll try it on a VM. I want to see how it works. I originally came here hoping I'd find code but nvm.
